### PR TITLE
Initial structural work for the google data logger.

### PR DESCRIPTION
**Pull request by mikehaney24**
Initial structural work for the google data logger.

### CODE REVIEW

File: GoogleDataLogger.podspec [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  s.ios.deployment_target = '8.0'
+
+  s.cocoapods_version = '>= 1.4.0'
+
+  # TODO(mikehaney24): Change to static framework after cocoapods 1.6.0 release?
+  s.static_framework = false
+  s.prefix_header_file = false
+
+  s.source_files = 'GoogleDataLogger/GoogleDataLogger/**/*'
+  s.public_header_files = 'GoogleDataLogger/GoogleDataLogger/Public/*.h'
```
Comment (paulb777) [inline GoogleDataLogger.podspec]: Comment out or delete until the files exist to pass pod lib lint

File: GoogleDataLogger.podspec [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  s.license          = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+  s.source           = {
+    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+    :tag => 'GoogleDataLogger-' + s.version.to_s
+  }
+
+  s.ios.deployment_target = '8.0'
+
+  s.cocoapods_version = '>= 1.4.0'
```
Comment (paulb777) [inline GoogleDataLogger.podspec]: Require 1.5.0 or maybe 1.5.3 for test specs.

File: GoogleDataLogger/.gitignore [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# OS X
```
Comment (paulb777) [inline GoogleDataLogger/.gitignore]: Delete file and use the one in top level directory.

File: GoogleDataLogger/.travis.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# references:
```
Comment (paulb777) [inline GoogleDataLogger/.travis.yml]: Use ../.travis.yml

File: GoogleDataLogger/GoogleDataLogger/Classes/GDLLogEvent.h [hunk 5]
```diff
@@ -0,0 +1,21 @@
+/*
```
Comment (paulb777) [inline GoogleDataLogger/GoogleDataLogger/Classes/GDLLogEvent.h]: Delete the two .gitkeep files.

File: GoogleDataLogger/generate_project.sh [hunk 6]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (paulb777) [inline GoogleDataLogger/generate_project.sh]: Put or point to pod generate instruction in the README - as will as a brief summary of what the project is and how to develop it.

File: GoogleDataLogger.podspec (hunk 1 above)
Comment (mikehaney24) [reply GoogleDataLogger.podspec]: The path was just wrong. I meant it to be GDL/GDL/Classes/Public/*.h

File: GoogleDataLogger/generate_project.sh [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
```
Comment (paulb777) [inline GoogleDataLogger/generate_project.sh]: Add copyright here too

**Pull request merged**